pkg/manifests/manifests.go

```diff
@@ -1410,6 +1411,17 @@ func (f *Factory) TelemeterClientClusterRoleBinding() (*rbacv1beta1.ClusterRoleB
 	return crb, nil
 }
 
+// TelemeterClientClusterRoleBindingView generates a new ClusterRoleBinding for Telemeter client
+// for the cluster monitoring view ClusterRole.
+func (f *Factory) TelemeterClientClusterRoleBindingView() (*rbacv1beta1.ClusterRoleBinding, error) {
+	crb, err := f.NewClusterRoleBinding(MustAssetReader(TelemeterClientClusterRoleBindingView))
+	if err != nil {
+		return nil, err
```

Comment:
unrelated: we seem to be inconsisten in the usage of `errors.Wrap`. Here we don't, but below we do wrap errors.

Reply:
that's correct. We could do this consistently for everything. It's mechanical and lazy, but would be nice.